README.mkd

```diff
@@ -257,12 +257,11 @@ greatly appreciated.
 
 ### Forks not yet fully integrated
 
-- [libogc/WPAD/DevKitPro](http://wiibrew.org/wiki/Libogc)
-  - Started before the disappearance of the original upstream
+- [libogc/wiiuse](https://github.com/devkitPro/libogc/tree/master/wiiuse)
+  - wii port created by Shagkur and contributed upstream
   - Focused on Wiimote use with Wii hardware
-  - Functions renamed, copyright statements removed
+  - code unfortunately diverged
   - Additional functionality unknown?
-  - git-svn mirror found here: <https://github.com/xloem/libogc-wiiuse>

```

Comment:
I suppose this is still in the git history, but part of me is wondering if it makes sense to leave that link and just re-label it? Though, given the fact that I had put this under "not yet fully integrated", perhaps I didn't actually pick any commits from there after all.